src/frontend/src/pages/WorkPackageDetailPage/ActivateWorkPackageModalContainer/ActivateWorkPackageModal.tsx

```diff
@@ -75,13 +74,17 @@ const ActivateWorkPackageModal: React.FC<ActivateWorkPackageModalProps> = ({
                 render={({ field: { onChange, value } }) => (
                   <>
                     <Typography>Project Lead</Typography>
-                    <Select onChange={onChange} value={value} variant="outlined" size="small" fullWidth>
-                      {allUsers.map((p) => (
-                        <MenuItem key={p.userId} value={p.userId}>
-                          {fullNamePipe(p)}
-                        </MenuItem>
-                      ))}
-                    </Select>
+                    <NERAutocomplete
+                      id="project-lead-autocomplete"
+                      onChange={onChange}
```

Comment:
oop also this doesnt work, you need to add the value={value} thing i think. Youll need to do the same thing for the other one.

Reply:
oop Actually what you need to do is change the form input to take in the object { label: string, id: string } and then when you make the request extract the id from it cause we want the user id yes yes yes